cdap-examples/WordCount/src/main/java/co/cask/cdap/examples/wordcount/WordCounter.java

```diff
@@ -41,15 +41,21 @@
  */
 public class WordCounter extends AbstractFlow {
 
+  private final WordCount.WordCountConfig config;
+
+  public WordCounter(WordCount.WordCountConfig config) {
+    this.config = config;
```

Comment:
ok i see. Do we have the property stuff documented anywhere? If not can you open a jira? it is not very intuitive.